src/pages/Settings/TwoStepFaq.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<div>
+		<kv-settings-card>
+			<template v-slot:content>
+				<div class="strong">
```

Comment:
Semantically this header is probably an <h2>

Reply:
I've updated these to h2, worth noting that it looks different from the mock because of this update.

Reply:
No problem there, we can always change the appearance with CSS to match designs if needed. It's important to use the right tags so that machines without eyes (like search engines and screen readers) can understand what the page _means_. Visuals gets layered on after that.